cds-snc/cppd-medical-report
# Style treatments screen and Delete treatment

# Description

NOTE: this is based off #69, merge that one first

Style the `treatments` list screen and enable deletes

# Test Instructions

- Create some conditions, some medications, and some treatments
- Observe the new treatments screen
- Try deleting one!
